sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -209,6 +231,11 @@ src_prepare() {
 	if use cuda; then
 		cuda_add_sandbox -w
 		cuda_src_prepare
+
+		get_cuda_arch
+		einfo "Using CUDA architecture ${MY_CUDA_ARCH}"
+		einfo "If auto detection fails for you, please try and export the CUDA_ARCH"
+		einfo "environment variable to one of the common arch names"
```

Comment:
Just to be pedantic, this seem like src_configure material rather than src_prepare (it's just picking a configure option).

If use src_configure and no get_cuda_arch function, could use a local variable rather than MY_CUDA_ARCH too, all contained within src_configure's cuda block.

Reply:
I mainly used src_prepare to fail as early as possible if anything goes wrong with this and to give the user a hint early on.

Reply:
Why would get_cuda_arch fail? It's just checking variables and giving info.

Also src_prepare makes no real difference, if you want to fail early it'd be pkg_pretend.